src/audio_call.c

```diff
@@ -958,24 +907,67 @@ void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)
         goto on_error;
     }
 
+    call->audio_bit_rate = bit_rate;
+
     return;
 
 on_error:
     print_err(self, error_str);
 }
 
+void place_call(ToxWindow *self)
+{
+    Call *call = &CallControl.calls[self->num];
+
+    if (call->status != cs_Pending) {
+        return;
+    }
+
+    Toxav_Err_Call error;
+    const char *error_str;
+
+    toxav_call(CallControl.av, self->num, call->audio_bit_rate, call->video_bit_rate, &error);
```

Comment:
`video_bit_rate` doesn't exist when we build for `AUDIO` only (`!VIDEO`). Either make the existence of that unconditional or avoid its use when `!VIDEO`.